File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int lines;
    //Solicita a informação de quantas linhas
    do
    {
        lines =  get_int("How many lines (1 - 8)?\n");
    } 
    while (lines < 1 || lines > 8);
    
    int col = 1;
    int lin;
    //imprir a piramide    
    for (lin = 1; lin < (lines + 1); lin++)
    {
        col = 1;
        //Imprimir espaços anteriores
        while (col <= (lines - lin))
        {
            printf(" ");
            col++;
        }              
        //imprimir as #
        for (; col <= lines; col++)
        {
            printf("#");
        }
        printf("  "); //imprimir os dois espacos entre as colunas
        for (col = 1; col < (lin + 1); col++)
        {
            printf("#"); //imprimir a parte final
        }
        printf("\n");
    }

}

File: pset6/mario.py
from cs50 import get_int
lines = 10
# get the lines argument
while lines <= 0 or lines > 8:
    lines = get_int("How many Lines (1 - 8)\n")

lin = 1
# loop of the number of lines
while lin < (lines + 1):
    col = 1
    # print the initials spaces
    while col <= (lines - lin):
        print(" ", end="")
        col = col + 1
    # Print initial #
    while col <= lines:
        print("#", end="")
        col = col + 1
    print("  ", end="")
    col = 1
    # Print the finals #
    while col < (lin + 1):
        print("#", end="")
        col = col + 1
    print()
    lin = lin + 1